hw/system/snitch_cluster/Makefile

```diff
@@ -126,11 +127,43 @@ sw/test.vlt: sw/build.vlt
 clean-sw:
 	rm -rf sw/build
 
+############################
```

Comment:
This _can_ be added if necessary after some cleanup, but it is cleaner to add these targets to in the same `riscv_tests`-specific directory as the rules you added in the top directory.